docs/services/slam/_index.md

```diff
@@ -31,3 +31,203 @@ Integrated SLAM Libraries include:
+**Returns:**
+
+- [(Pose)](https://python.viam.dev/autoapi/viam/components/arm/index.html#viam.components.arm.Pose): A `Pose` representing the current position of the component the SLAM service is configured to source point cloud data from.
+For example, a [camera](/components/camera/) named `"cam"`.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/slam/client/index.html#viam.services.slam.client.SLAMClient.get_position).
+
+```python {class="line-numbers linkable-line-numbers"}
+slam_svc = SLAMClient.from_robot(robot=robot, name="my_slam_service")
+
+# Get the current position of the specified source component in the SLAM map as a Pose.
+pose = await slam.get_position()
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetPointCloudMap
+
+Get the point cloud map.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- (func() [[]byte](https://pkg.go.dev/builtin#byte), [error](https://pkg.go.dev/builtin#error)): The complete point cloud map in standard PCD format.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/slam#Service).
+
+```go {class="line-numbers linkable-line-numbers"}
+slam_svc, err := slam.FromRobot(robot, "my_slam_service")
+
+// Get the point cloud map in standard PCD format.
+pcd_map, err := slam_svc.GetPointCloudMap(context.Background())
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(List[GetPointCloudMapResponse])](https://python.viam.dev/autoapi/viam/gen/service/slam/v1/slam_pb2/index.html#viam.gen.service.slam.v1.slam_pb2.GetPointCloudMapResponse): The complete point cloud map in standard PCD format.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/slam/client/index.html#viam.services.slam.client.SLAMClient.get_point_cloud_map).
+
+```python {class="line-numbers linkable-line-numbers"}
+slam_svc = SLAMClient.from_robot(robot=robot, name="my_slam_service")
+
+# Get the point cloud map in standard PCD format.
+pcd_map = await slam.get_point_cloud_map()
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetInternalState
+
+Get the internal state of the SLAM algorithm required to continue mapping/localization.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
```

Comment:
@sguequierre should "extra" be here? It's not in the [slam.proto](https://github.com/viamrobotics/api/blob/03915ccb108a2284ea0083edc8c8edffc4f825b0/proto/viam/service/slam/v1/slam.proto#L97-L100) and in the example below GetInternalState just takes a `ctx` argument

Reply:
no you're absolutely right, I did not catch this when trying to QA yesterday-- thank you!

Reply:
will fix